admin/app/controllers/authenticated/certifications/certification/details.js

```diff
@@ -57,7 +57,7 @@ export default class CertificationDetailsController extends Controller {
         return marks;
       }, {})
     });
-    this.transitionToRoute('authenticated.certifications.certification.informations', this.get('details.id'));
+    this.transitionToRoute('authenticated.certifications.certification.informations', this.details.id);
```

Comment:
```suggestion
    this.transitionToRoute('authenticated.certifications.certification.informations', this.details.get('id'));
```

Reply:
Vu ensemble : this.details est un alias du model
Done